chainbench/test_data/base.py

```diff
@@ -78,6 +105,18 @@ def data(self) -> BlockchainData:
 
         return self._data
 
+    @staticmethod
+    def _parse_hex_to_int(value: str) -> int:
+        return int(value, 16)
```

Comment:
Do you really need this function? `self._parse_hex_to_in(var)` is longer than `int(var, 16)`

Reply:
This function was created by Egor Molodik. I think his principle in coding is to make code simple to read and understand. So for anyone who is looking at the code, they will know what it is for.